dotnet-authserver/tests/TeacherIdentity.AuthServer.Tests/EndpointTests/SignIn/Trn/CheckAnswersTests.cs

```diff
@@ -31,21 +31,13 @@ public async Task Get_JourneyHasExpired_RendersErrorPage()
         await JourneyHasExpired_RendersErrorPage(c => c.OfficialNameSet(), HttpMethod.Get, "/sign-in/trn/check-answers");
     }
 
-    [Theory]
```

Comment:
I split up this test into a bunch of smaller ones as it was becoming unwieldy with the added logic.